scripts/dotnet-cli-build/MsiTargets.cs

```diff
@@ -60,6 +69,20 @@ private static void AcquireWix(BuildTargetContext c)
             ZipFile.ExtractToDirectory(Path.Combine(WixRoot, "WixTools.zip"), WixRoot);
         }
 
+        private static void AcquireCrt(BuildTargetContext c)
+        {
+            if (File.Exists(CrtRedistributablePath))
+            {
+                return;
+            }
+
+            Directory.CreateDirectory(Path.GetDirectoryName(CrtRedistributablePath));
+
+            c.Info($"Downloading VC_redist.{Arch}.{CrtVersion}.exe..");
+
+            DownloadFile($"https://dotnetcli.blob.core.windows.net/build/vc/{Arch}/VC_redist.{Arch}.{CrtVersion}.exe", CrtRedistributablePath);
```

Comment:
Instead of this, you could consider just letting the bundle always download the VC Redist from this location or MSFT (depending which you trust more). Given most of your customers probably already have VC Redist installed, the download will usually get skipped and you won't need to have the bits as part of the build, just use the `RemotePayload` element.